bazel.el

```diff
@@ -88,8 +88,25 @@
           "https://github.com/bazelbuild/buildtools/tree/master/buildifier")
   :risky t)
 
-;;;; Commands to run Buildifier.
+(defconst bazel--magic-comment-regexp
+  (rx (or "keep sorted"
+          "do not sort"
+          "@unused"
+          "@unsorted-dict-items"
+          "buildifier: leave-alone"
+          (seq (or "buildifier" "buildozer") ": "
+               (seq "disable=" (+ (any "A-Za-z-"))))))
```

Comment:
This additional `seq` shouldn't be necessary.

Reply:
Indeed it is not.  Removed.